eng/Directory.Build.Common.props

```diff
@@ -142,7 +142,8 @@
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <Authors>Microsoft</Authors>
     <Product>Azure .NET SDK</Product>
-    <PackageIconPath>$(RepoEngPath)/images/pkgicon.png</PackageIconPath>
+    <PackageIcon>pkgicon.png</PackageIcon>
+    <PackageIconPath>$(RepoEngPath)/images/pkgicon.png</PackageIconPath>  
```

Comment:
```suggestion
    <PackageIconPath>$(RepoEngPath)/images/$(PackageIcon)</PackageIconPath>  
```

Reply:
I initially did it this way, but then it begs the question of why not factor out this property instead of adding it twice, which means we'd need to add a new property group just for this property. Rather than go down that path, it seemed simpler to just hardcode it. I agree that we can make the change you suggest, but the way it is now seems more clear in my head.

Reply:
@weshaggard If you feel strongly about it, I can commit this change. I'll just say that I prefer the way it is now.

Reply:
I don't have a strong preference I just don't like repeating things where a mistake can easily be made.

Reply:
That's fair - I ended up committing the suggestions.